doc/system-telemetry/reboot-cause.md

```diff
@@ -11,78 +11,73 @@
 
 ### Enable sonic streaming telemetry agent to send Reboot-cause information
 
-##### Part 1 
-For 1st part, process-reboot-cause copies the previous-reboot-cause.txt 
-to "/host/reboot-cause/previous-reboot-cause/" with adding timestamp 
-at the end of file name after processing the reboot-cause.
+#### Part 1
+During the boot, the process-reboot-cause processes the last reboot-cause based on the hardware reboot-cause
+and the software reboot-cause information and creates previous-reboot-cause.txt with the information.
+To log upto 10 entries of the previous reboot-cause, `process-reboot-cause` will save the previous reboot cause information
+to "/host/reboot-cause/previous-reboot-cause/" with adding timestamp at the end of file name after processing
+the reboot-cause and create a symbolic link.
```

Comment:
Can you add more details about what this symbolic link is used for, its name, location, etc.?